Missing dependency ctypes : oasis ?

```
==== ERROR [while installing ctypes.0.1] ====
# opam-version    1.0.0
# os              darwin
# command         make
# path            /Users/mathieubaudet/.opam/4.00.1/build/ctypes.0.1
# exit-code       2
# env-file        /Users/mathieubaudet/.opam/4.00.1/build/ctypes.0.1/ctypes-c61e7b.env
# stdout-file     /Users/mathieubaudet/.opam/4.00.1/build/ctypes.0.1/ctypes-c61e7b.out
# stderr-file     /Users/mathieubaudet/.opam/4.00.1/build/ctypes.0.1/ctypes-c61e7b.err
### stdout ###
oasis setup
### stderr ###
make: oasis: No such file or directory
make: *** [setup.ml] Error 1
```

--- a/packages/ctypes.0.1/opam
+++ b/packages/ctypes.0.1/opam
@@ -7,4 +7,4 @@ build: [
 remove: [
   ["ocamlfind" "remove" "ctypes"]
 ]
-depends: [ "ocamlfind" "ounit" ]
+depends: [ "ocamlfind" "ounit" ("oasis" {>="0.3.0"} |"oasis-mirage" {>="0.3.0"}) ]

